tests/queries/0_stateless/01273_arrow_stream.reference

```diff
@@ -0,0 +1,62 @@
+9999
+9998
+9997
+9996
+9995
+9994
+9993
+9992
+9991
+9990
+99999
+99998
+99997
+99996
+99995
+99994
+99993
+99992
+99991
+99990
+2
+1
+0
+999
+998
+997
+996
+995
+994
+993
+992
+991
+990
+ContextLock	Number of times the lock of Context was acquired or tried to acquire. This is global lock.
```

Comment:
Let's remove this text from test reference -- or the test will break if we decide to improve the description.

By the way, we recently added a table function to reproducibly generate pseudo-random tables: `generateRandom`. See https://clickhouse.tech/docs/en/sql-reference/table-functions/generate/